libraries/lib-components/EffectInterface.h

```diff
@@ -80,9 +80,16 @@ class COMPONENTS_API EffectSettingsExtra final {
    //! @return value is not negative
    double GetDuration() const { return mDuration; }
    void SetDuration(double value) { mDuration = std::max(0.0, value); }
+
+   //! Versioning counter for detecting echo from worker thread;
+   //! it does not need a large range of values
+   using Counter = unsigned char;
+   Counter GetCounter() const { return mCounter; }
+   void SetCounter(Counter value) { mCounter = value; }
 private:
    NumericFormatSymbol mDurationFormat{};
    double mDuration{}; //!< @invariant non-negative
+   Counter mCounter{ 0 };
```

Comment:
The assumption is that it's unlikely that there will be 255 calls to `Set` between two `Get`'s?

Reply:
Yes:  the point of this commit is something like a request/response alternation between the main and worker thread.

Reply:
The counter might even be just one bit